repo2docker/buildpacks/conda/__init__.py

```diff
@@ -293,9 +294,9 @@ def get_env_scripts(self):
                 (
                     "${NB_USER}",
                     r"""
-                conda install -p {0} r-base{1} r-irkernel={2} r-devtools && \
-                conda clean --all -f -y && \
-                conda list -p {0}
+                mamba install -p {0} r-base{1} r-irkernel={2} r-devtools -y && \
```

Comment:
Ping @wolfv 

For some readon, `conda` did not require passing `-y` here.

Reply:
is there a conda-rc? or environment var that could enable this behavior automagically?

Reply:
maybe conda detects TTY with `sys.stdin.isatty()`? That's a pretty common pattern for deciding if a program can interact with the terminal or not.

Reply:
yeah we do have that mechanism as well, but currently don't use it to set auto_yes = True.

Reply:
Are we sure this is what is going on?